content/en/docs/releases/feature_lifecycle.md

```diff
@@ -104,4 +104,16 @@ foo: bar
 
 The `helm show values` command can be used to view the Helm chart's values.
 
-A value marked as `DEPRECATED` in a release will only be removed in the next minor release at the earliest. This means, a value deprecated in release `vX.Y.Z` will only be removed on or after release `vX.Y+1.Z`. Deprecated values will be documented in the release notes.
\ No newline at end of file
+A value marked as `DEPRECATED` in a release will only be removed in the next minor release at the earliest. This means, a value deprecated in release `vX.Y.Z` will only be removed on or after release `vX.Y+1.Z`. Deprecated values will be documented in the release notes.
+
+## API Deprecation
+
+Similar to the [Kubernetes API deprecation policy](https://kubernetes.io/docs/reference/using-api/deprecation-policy/), GA or stable APIs will be marked as deprecated prior to removal.
+
+Stable APIs marked as deprecated:
+
+- Remain functional but will generate a warning if used
+- Will not be removed within a major release version
```

Comment:
will these also be removed in the next minor release then?

Reply:
I should clarify, the intention was to say the earliest a deprecated feature would be removed is the next major release version. Thank you @SanyaKochhar

Reply:
fixed

Reply:
great thanks!